packages/trader/src/App/Components/Elements/__tests__/chart-loader.spec.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ChartLoader from '../chart-loader';
+
+describe('ChartLoader', () => {
+    const test_id = 'dt_barspinner';
+
+    it('should render ChartLoader component if is_visible  === true', () => {
+        render(<ChartLoader is_visible />);
+
+        expect(screen.getByTestId(test_id)).toBeInTheDocument();
+    });
+    it('should render ChartLoader component if is_visible  === false', () => {
+        render(<ChartLoader />);
```

Comment:
```suggestion
        render(<ChartLoader is_visible={false} />);
```
Doesn't the component is shown on the basis of is_visible props and if yes then what is the default value?

Reply:
@hamza-deriv a default value is not set in ChartLoader for is_visible prop, that's why is_visible will be falsy (undefined), if we don't pass is_visible into  ChartLoader. 🙂 So we can omit `is_visible={false}` here

Reply:
Hi @hamza-deriv ! Thank you for comment👍🏻
Component `<ChartLoader />` render child component `<Loading />` based on `is_visible` prop. If it's equal to `true`, then it renders `<Loading />` , if not - then `null` (please, refer to _chart-loader.jsx_, line 10). 
`is_visible` has no default value, so in first test case I checked whether it renders child component (`<Loading />` has `test_id = 'dt_barspinner'`) and in the 2d case I didn't pass `is_visible `(so it's equal to `undefined`) and expect NOT to find child's test-id (as it shouldn't be rendered)

Reply:
Thanks for the explanantion